pkgs/tools/security/ibm-sw-tpm2/default.nix

```diff
@@ -12,28 +11,18 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "ibm-sw-tpm2";
-  version = "1682";
+  version = "rev183-2024-08-02";
```

Comment:
The convention here would be `1682-unstable-2024-08-02`, as silly as that is for this particular versioning scheme.

Reply:
Thanks for the suggestion. Applied.